inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -452,6 +571,15 @@ void ngfunction_2_irv10(pugi::xml_document& doc,
                 NGRAPH_CHECK(i.get_partial_shape().is_static(),
                              "Unsupported dynamic input shape in ", node);
 
+                // WA for LSTMCellv0, peephole input shall not be serialized
+                if (i.get_index() == 6) {
+                    auto type_info = node->get_type_info();
+                    if (!strcmp(type_info.name, "LSTMCell") && type_info.version == 0) {
+                        port_id++;
```

Comment:
Not sure that we need to increment port_id and not just skip.  Could you please explain?

Reply:
If port_id incrementation is skipped then we need to align back_edges with newer port numeration. It will complicate WA implementation for LSTMCell.